docs/semgrep-assistant/overview.md

```diff
@@ -124,7 +124,6 @@ Semgrep uses API permissions to access code on your pre-selected GitHub or GitLa
 * Semgrep, Inc. is a paying customer of OpenAI and has a Data Protection Agreement signed with them (provided upon request by [contacting support](/docs/support). The code snippets we upload are persisted by OpenAI temporarily, following their data usage policies at [Enterprise privacy at OpenAI](https://openai.com/enterprise-privacy).
 <!-- markdown-link-check-enable -->
 * Semgrep, Inc. takes the following steps to protect data that is processed by AI since Assistant requires the sharing of code snippets with a third party:
-  * Semgrep shares code snippets with OpenAI without identifying the customer or repository name.
```

Comment:
@jsmoxon What exactly has changed here - do we share code snippets with openai at all? is it possible to at least keep part of this bullet (let's say we don't identify the customer, but we do send the repository name)?

Reply:
We do share code snippets with OpenAI. What has changed is that we now sometimes share the repository name. And sometimes it is possible to determine the customer name from the repository name. E.g. if the repo is called 'acme-foo-service`. 

Previously we didn't share the repo name. @Sly1029 correct me if I have this wrong.

Reply:
Yeah now we have some desire to start sharing this information. But I wanted to leave it up to Jack to do the comms first

Reply:
Ah yeah that second option sounds good